src/mumble/IPCUtils.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_IPCUTILS_H_
+#define MUMBLE_MUMBLE_IPCUTILS_H_
+
+#include <QtCore/QDir>
+
+namespace Mumble {
+
+	/**
+	 * @returns The path where Mumble sets up MumbleOverlayPipe and MumbleSocket
+	 */
+	QDir getRuntimeDir(void);
```

Comment:
```suggestion
	QDir getRuntimeDir();
```
Imo this function should return a String to not even add the `QDir`  dependency for it. You can always convert `std::string` to a `QDir`, if that's what you need.